analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -845,22 +678,9 @@ def get_query(source: ObjectStream) -> ObjectStream:
 
 # %%
 model_prediction = cabinetry.model_utils.prediction(model)
-figs = cabinetry.visualize.data_mc(model_prediction, data, close_figure=True, config=config)
-figs[0]["figure"]
-
-# %% tags=[]
-figs[1]["figure"]
-
-# %% [markdown]
-# We can see very good post-fit agreement.
-
-# %% tags=[]
 model_prediction_postfit = cabinetry.model_utils.prediction(model, fit_results=fit_results)
-figs = cabinetry.visualize.data_mc(model_prediction_postfit, data, close_figure=True, config=config)
-figs[0]["figure"]
-
-# %% tags=[]
-figs[1]["figure"]
+figs = cabinetry.visualize.data_mc(model_prediction, data, close_figure=True, config=config)
```

Comment:
This now effectively only does the saving to file, right? Since the line below does the actual visualization, it does not save too much of a purpose anymore. Let's add a comment to the next line then saying that this re-implements the visualization in a grid view (hopefully we can eventually replace that, but there's an issue already to track this anyway).